moped-database/views/component_arcgis_online_view.sql

```diff
@@ -191,7 +191,6 @@ SELECT
     plv.interim_project_id,
     plv.project_partners,
     plv.task_order_names,
-    plv.funding_source_name,
```

Comment:
still loving how helpful it is to get these diffs on views

Reply:
sameee